packages/block-library/src/template-part/edit/index.js

```diff
@@ -131,17 +130,15 @@ export default function TemplatePartEdit( {
 								contentClassName="wp-block-template-part__preview-dropdown-content"
 								position="bottom right left"
 								renderToggle={ ( { isOpen, onToggle } ) => (
-									<ToolbarButton
+									<Button
```

Comment:
Is there a reason we are replacing `ToolbarButton` with `Button`?  I think the idea of ToolbarButton component is to have consistent styling and keyboard interactions across the toolbar.

Reply:
I couldn't work out how to get the ToolbarButton to display text 🙈

Reply:
It looks like it works the same way here with the text child element `{ __( 'Replace' ) }`.

I hope you don't mind, but I updated to use the ToolbarButton and rebased the PR to handle the conflicting files.  It seems to be working well, but let me know if there are any issues. 😁

Reply:
Of course I don't mind :D Thanks so much.

Reply:
Awesome.  Can you confirm the recent changes also test well on your end?